project/plugins.sbt

```diff
@@ -3,6 +3,4 @@ addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.4.1")
 // Needed for importing the project into Eclipse
 addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.2.4")
 
-addSbtPlugin("com.typesafe.sbt"    % "sbt-native-packager" % "1.1.6")
-
-addSbtPlugin("com.lightbend.rp"    % "sbt-reactive-app"    % "0.6.1")
```

Comment:
Only use `sbt-reactive-app` for Kubernetes for now. DC/OS needs the individual components wired together.